modules/osint/domain_reputation.py

```diff
@@ -24,10 +24,10 @@
 	'author': 'Kunal Khandelwal',
 	'version': '0.1',
```

Comment:
change the version

Reply:
@saeeddhqan Will the underscores be ok then?
```
[*] ABSENCE_%_ON_BLACKLIST
[*] 	100.0

[*] NORTON_SAFE_WEB_REPORT:
[*] 	Threat Report: zenithsecurity.org is a known dangerous web page
```

Reply:
put the % in value instead of key. the key should be exactly like a username